File: customers-staff-master/src/Front/src/api/new/dto/CustomerInfoDto.ts
import { GenderType } from "../../../models/GenderType";
import { BirthdayDto } from "./BirthdayDto";

export interface CustomerInfoDto {
  name?: string;
  birthday?: BirthdayDto;
  phone?: string;
  email?: string;
  customId?: string;
  discount?: number;
  gender?: GenderType;
  additionalInfo?: string;
}

File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/Customers/CustomerInfoDto.cs
namespace Market.CustomersAndStaff.FrontApi.Dto.Customers
{
    public class CustomerInfoDto
    {
        public string Name { get; set; }
        public BirthdayDto Birthday { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string CustomId { get; set; }
        public decimal? Discount { get; set; }
        public GenderDto? Gender { get; set; }
        public string AdditionalInfo { get; set; }
    }
}